Assets/HexMap.cs

```diff
@@ -215,7 +188,9 @@ public Hex GetAcceptableStartPosition()
             int impassibleTiles = 0;
             foreach (Hex h in neighbors)
             {
-                if (h.GetTerrain() == Terrain.MOUNTAIN || h.GetTerrain() == Terrain.WATER)
+                if (h == null)
+                    Debug.LogWarning("Null neighbor, something is borked");
```

Comment:
The debug warning. There is sometimes a smell in checking for null, but that seems too nitpicky at this stage. I would think you should be throwing and handling an error further up the chain and logging it, but maybe not. I am not sure of the best practice in a unity project.